exporters/otlp-http/metrics/src/main/java/io/opentelemetry/exporter/otlp/http/metric/OtlpHttpMetricExporter.java

```diff
@@ -0,0 +1,199 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.exporter.otlp.http.metric;
+
+import com.google.rpc.Code;
+import com.google.rpc.Status;
+import io.opentelemetry.exporter.otlp.internal.MetricAdapter;
+import io.opentelemetry.proto.collector.metrics.v1.ExportMetricsServiceRequest;
+import io.opentelemetry.sdk.common.CompletableResultCode;
+import io.opentelemetry.sdk.internal.ThrottlingLogger;
+import io.opentelemetry.sdk.metrics.data.MetricData;
+import io.opentelemetry.sdk.metrics.export.MetricExporter;
+import java.io.IOException;
+import java.util.Collection;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+import javax.annotation.Nullable;
+import javax.annotation.concurrent.ThreadSafe;
+import okhttp3.Call;
+import okhttp3.Callback;
+import okhttp3.Headers;
+import okhttp3.MediaType;
+import okhttp3.OkHttpClient;
+import okhttp3.Request;
+import okhttp3.RequestBody;
+import okhttp3.Response;
+import okhttp3.ResponseBody;
+import okio.BufferedSink;
+import okio.GzipSink;
+import okio.Okio;
+
+/** Exports metrics using OTLP via HTTP, using OpenTelemetry's protobuf model. */
+@ThreadSafe
+public final class OtlpHttpMetricExporter implements MetricExporter {
+
+  private static final MediaType PROTOBUF_MEDIA_TYPE = MediaType.parse("application/x-protobuf");
+
+  private final ThrottlingLogger logger =
+      new ThrottlingLogger(Logger.getLogger(OtlpHttpMetricExporter.class.getName()));
+
+  private final OkHttpClient client;
+  private final String endpoint;
+  @Nullable private final Headers headers;
+  private final boolean isCompressionEnabled;
```

Comment:
how about just calling this `compressionEnabled` without the `is`?

Reply:
Yeah as long as there's no convention, I prefer that. Will do that to the equivalent fields over in `OtlpHttpSpanExporter` as well.